scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz

```diff
@@ -56,7 +56,7 @@ RUN git clone -b boost-1.69.0 https://github.com/boostorg/boost.git \
     rm -rf /usr/src/boost
 
 # Z3
-RUN git clone --depth 1 -b z3-4.8.7 https://github.com/Z3Prover/z3.git \
+RUN git clone --depth 1 -b z3-4.8.9 https://github.com/Z3Prover/z3.git \
```

Comment:
I wonder why it didn't fail if it was still at 4.8.7.

Reply:
Because that job doesn't run `soltest`, just `solfuzzer`, so the error expectation is not checked.

Reply:
I also wonder whether the google image is actually still based on ubuntu 16.04 at all, or whether they moved on by now :-)... difficult to tell just from the file here :-). Not that it matters too much anyways, it's just a name...